docs/reference-guides/data/data-core-block-editor.md

```diff
@@ -1493,10 +1493,10 @@ _Parameters_
 
 -   _clientId_ `string`: Block client ID.
 
-### selectPreviousBlock
+### selectPreviousBlockOrFirstParent
```

Comment:
`selectPreviousBlock` is a public API and we can't just rename it. We should either create a new API or introduce a new parameter in the existing one.

Reply:
Oh yes 🤦🏻

Reply:
Solved in [fb6660f](https://github.com/WordPress/gutenberg/pull/48204/commits/fb6660f97acbfe6c2ab2f5017243cfe9b29719e3)